plugins/drivers/utils.go

```diff
@@ -412,7 +412,7 @@ func TaskStatsFromProto(pb *proto.TaskStats) (*TaskResourceUsage, error) {
 	}
 
 	stats := &TaskResourceUsage{
-		Timestamp:     timestamp.Unix(),
+		Timestamp:     timestamp.UnixNano(),
```

Comment:
Question for my edification - how did docker have this timestamp in nanoseconds if all drivers share this `TaskStatsfromProto` method for statistics?